packages/block-library/src/rss/block.json

```diff
@@ -46,6 +46,26 @@
 		"html": false,
 		"interactivity": {
 			"clientNavigation": true
+		},
+		"color": {
+			"background": true,
+			"text": true,
+			"gradients": true,
+			"link": true
+		},
+		"typography": {
+			"fontSize": true,
+			"lineHeight": true,
+			"__experimentalFontFamily": true,
```

Comment:
I think we can remove the `__experimental` prefixes now that https://github.com/WordPress/gutenberg/pull/63401 has merged 👍🏻

Reply:
+1

Reply:
__experimental prefixes removed in recent commit.